plugins/rest/rest_auth.go

```diff
@@ -18,6 +19,12 @@ import (
 	"strings"
 	"time"
 
+	"github.com/open-policy-agent/opa/internal/uuid"
+
+	"github.com/open-policy-agent/opa/keys"
+
+	"github.com/open-policy-agent/opa/internal/jwx/jws"
+
```

Comment:
Can these be put together in a third block, so we have
```
import (
    "fmt" // stdlib
     ...
     
     "github.com/foo/bar" // third party
     ...
     
     "github.com/open-policy-agent/opa/keys" // same repo
     ...
)
```

Reply:
Ah yeah looks like I need to tweak my editor settings with regards to imports. Thought the formatter would fix that but apparently not. Fixing 👍